packages/wrangler/src/index.ts

```diff
@@ -465,6 +460,15 @@ export function createCLIParser(argv: string[]) {
 		}
 	);
 
+	// vectorize
+	wrangler.command(
+		"vectorize",
+		"🧮 Interact with Vectorize indexes",
+		(vectorYargs) => {
+			return vectorize(vectorYargs.command(subHelp));
+		}
+	);
+
```

Comment:
This is my britishness talking, but I'd _love_ to see a hidden `vectorise` alias for this command 😅 (non-blocking, probably not a good idea)